front/components/items/Appointment.tsx

```diff
@@ -104,4 +110,15 @@ export default Appointment
 
 type filters = {
     [Keys: string]: Object,
+}
+
+type appointmentQueryType = {
```

Comment:
I'd create ts file gathering types in lib/strapi/appointments with this type. It'll be easier for future use. Moreover it's just my opinion but I believe that this typa could get more specific name. I'd try with strapiAppointmentType or maybe strapiAppointmentQueryType - for me its so much more important to know that you are reffering to strapi type than that this is some queryType. And last thing - its your convention, but im not so sure if you need this 'type' at the end of type. But even if I dont like this it doesnt mean that this is wrong typing